OMCompiler/Compiler/Util/Flags.mo

```diff
@@ -875,7 +875,7 @@ constant ConfigFlag TEARING_METHOD = CONFIG_FLAG(44, "tearingMethod",
   NONE(), EXTERNAL(), STRING_FLAG("cellier"),
   SOME(STRING_DESC_OPTION({
     ("noTearing", Gettext.gettext("Skip tearing.")),
```

Comment:
I would suggest "Skip tearing. This breaks models with mixed continuous-integer/boolean unknowns".

Otherwise this is not at all obvious, which is the reason why I opened the original ticket

Reply:
Done

Reply:
Very good, thanks!